packages/testing/package.json

```diff
@@ -28,12 +29,11 @@
     "nyc": "11.6.0",
     "rollup": "0.57.1",
     "rollup-plugin-commonjs": "9.1.0",
+    "rollup-plugin-copy-assets": "1.0.0",
     "rollup-plugin-node-resolve": "3.3.0",
-    "rollup-plugin-typescript2": "0.12.0"
-  },
-  "peerDependencies": {
```

Comment:
for testing the size savings are not important and this makes it slightly simpler for our clients